cmake/Modules/FindGPTL.cmake

```diff
@@ -0,0 +1,175 @@
+# FindGPTL.cmake
```

Comment:
I presume these are essentially copies of the ones in jedi-cmake, included to be consistent with the MPAS-dev upstream?

Reply:
Yes, but I'm not depending on jedi-cmake here, since we hope to keep the MPAS-Model itself idependent of JEDI since there are many other users and maybe this build system will eventually get merged upstream.